Unable to build on 32 bit system

I am running `make WEBCONSOLE=default` and it results in error `../go/pkg/mod/github.com/jaswdr/faker@v1.4.2/payment.go:26:41: constant 1000000000000000 overflows int`

What can I do? Can I not include data faker in webconsole?

--- a/go.mod
+++ b/go.mod
@@ -17,7 +17,7 @@ require (
 	github.com/huandu/xstrings v1.3.2 // indirect
 	github.com/imdario/mergo v0.3.12 // indirect
 	github.com/jackc/pgx/v4 v4.12.0
-	github.com/jaswdr/faker v1.4.2
+	github.com/jaswdr/faker v1.4.3
 	github.com/kr/pretty v0.2.0 // indirect
 	github.com/lib/pq v1.10.2
 	github.com/mattn/go-isatty v0.0.13 // indirect

